artemis/src/main/java/tech/pegasys/artemis/services/ServiceController.java

```diff
@@ -25,22 +28,24 @@
       ServiceFactory.getInstance(BeaconChainService.class).getInstance();;
   private static final PowchainService powchainService =
       ServiceFactory.getInstance(PowchainService.class).getInstance();
+  private static final P2PService p2pService =
+      ServiceFactory.getInstance(P2PService.class).getInstance();
   private static final ExecutorService beaconChainExecuterService =
       Executors.newSingleThreadExecutor();
   private static final ExecutorService powchainExecuterService =
       Executors.newSingleThreadExecutor();
+  private static final ExecutorService p2pExecuterService = Executors.newSingleThreadExecutor();
```

Comment:
I guess it's just under the column limit, so spotless didn't care, but let's try this for consistency.
```suggestion
  private static final ExecutorService p2pExecuterService =
      Executors.newSingleThreadExecutor();
```